### PR TITLE
chore(build): ephemeral rename, nav items

### PR DESCRIPTION
**Pull request by san7ket**
## What's included

...

## Example

...

### CODE REVIEW

**Comment by cdcabrera** [comment]
@san7ket few pieces
- commit message needs to be updated to reflect scope for tracking/changelog purposes, recommend something like
   - `chore(build): rename, openshift container` or `fix(build): ...`  if its an error fix. there's also a message length limit
- next, `dev`/feature is being planned for merge into `main` next week.  (merging into `main` at this time indicates a priority over the current work lined up for RHEL VDC)
   - As much as we enjoy rebasing feature work,  is this rename a make/break type scenario? 
   - Can this PR be redirected to point at `dev` instead?

**Comment by san7ket** [comment]
> @san7ket few pieces
> 
> * commit message needs to be updated to reflect scope for tracking/changelog purposes, recommend something like
>   
>   * `chore(build): rename, openshift container` or `chore(build): ephemeral rename, openshift` or `fix(build): ...`  if its an error fix. there's also a message length limit
> * next, `dev`/feature is being planned for merge into `main` next week.  (merging into `main` at this time indicates a priority over the current work lined up for RHEL VDC)
>   
>   * As much as we enjoy rebasing feature work,  is this rename a make/break type scenario?
>   * Can this PR be redirected to point at `dev` instead?

Thanks, for the comments, I don't think it is urgent, I thought the merge job would run on main branch, but I see that's not the case. I have updated the branch to dev and will improve the commit message, also I would like to use this as a testing PR to play around with left nav in EE, if that's possible. I will also mark it as draft until I can figure out a way to make the ephemeral EE as close to stage as possible.

**Comment by cdcabrera** [comment]
@san7ket that works for us! applied the "build:hold" label... if/when this is ready to go just give us/me a heads up

**Comment by san7ket** [comment]
We can merge this @cdcabrera

**Comment by san7ket** [comment]
@cdcabrera Any pointers as to why checks are failing

**Comment by cdcabrera** [comment]
@san7ket apologies we missed your note about merging. I can bring the PR up-to-date. 

Do you still want this merged?

**Comment by cdcabrera** [comment]
@san7ket we brought this PR up-to-date, however...
RHEL and OpenShift are covered now in the stage-beta environment where this PR would merge first... we're assuming this can be shut down now?

**about the pr failing, if curious**
- Looks like the GitHub CI cache was suffering from an outdated package failure... we just removed the cache files and restarted the job
- Also, the dist output testing checks against the yaml menu file for coverage for existing product configs in the repo. That check/snapshot needs to be updated every time the yaml file is adjusted around the menu... (it also happens to be a little hidden in the process)
   1. update the NPM packages with `$ yarn`
   2. run `$ yarn build` to get the failure and available files
   3. next run `$ yarn test:integration-dev` follow the directions to update the file
   4. finally commit any changes with your PR

**Comment by san7ket** [comment]
Both the PR's related to the PRchecks are blocked due to the new navigation (Business  --> Subscriptions)change that is implemented in the stage but the changes have not yet propagated to the frontend-operator (last time I checked ) for Ephemeral environments. The issue is being tracked at https://issues.redhat.com/browse/RHCLOUD-25569. I can recheck again this week if anything has changed in EE. Meanwhile, I think we should hold this and 1072 PR.

**Comment by san7ket** [comment]
@cdcabrera Tested this with latest code, we can merge this, this is needed to nav inbetween pages RHEL to openshift and vice versa in ephemeral envs.

**Comment by cdcabrera** [comment]
@san7ket much appreciated on the heads up!

we'll go ahead and rebase this and get it merged today 20230815

File: deploy/frontend.yaml
Comment (cdcabrera) [inline deploy/frontend.yaml:48]: apologies, almost missed this... 2 questions @san7ket 

- There appears to be a `RHEL` option that is duplicated around line 27-30, is this intentional?
- And we just removed `RHOSAK` or `streams` do we also want to remove it here?

Comment (san7ket) [reply deploy/frontend.yaml:48]: We want to keep "RHEL" and "RHEL Subscriptions": , ephemeral UI is not intelligent so we need to mentioned both, as our tests needs both to be present.

RHOSAK we will drop if we have already dropped from stage

Comment (cdcabrera) [reply deploy/frontend.yaml:48]: Yes, RHOSAK was dropped from prod and stage as of the GUI release last week, [v3.2.0](https://github.com/RedHatInsights/curiosity-frontend/blob/main/CHANGELOG.md#320-2023-08-15), so if you remove it that should be the last change

**Review by cdcabrera — APPROVED** [review]

**Comment by cdcabrera** [comment]
Looks like I need to double check something on the integration check before we merge...

**Review by cdcabrera — APPROVED** [review]

**Pull request merged**
